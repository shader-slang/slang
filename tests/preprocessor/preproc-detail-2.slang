//DIAGNOSTIC_TEST:SIMPLE:-E

// Macro parameters must have unique names

#define A(x, x) a x b x c

A(,)

