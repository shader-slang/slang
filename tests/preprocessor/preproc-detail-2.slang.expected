result code = 0
standard error = {
tests/preprocessor/preproc-detail-2.slang(7): error 15408: redefinition of macro parameter 'A'
#define A(x, x) a x b x c
             ^
}
standard output = {
a b c 
}
